seatunnel-connectors-v2/connector-iotdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/iotdb/source/IoTDBSource.java

```diff
@@ -64,7 +66,11 @@ public void prepare(Config pluginConfig) throws PrepareFailException {
             result = CheckConfigUtil.checkAllExists(pluginConfig, NODE_URLS.key());
 
             if (!result.isSuccess()) {
-                throw new PrepareFailException(getPluginName(), PluginType.SOURCE, "host and port and node urls are both empty");
+                throw new IotdbConnectorException(SeaTunnelAPIErrorCode.CONFIG_VALIDATION_FAILED,
+                    String.format("PluginName: %s, PluginType: %s, Message: %s",
+                        getPluginName(), PluginType.SOURCE,
+                        "host and port and node urls are both empty")
```

Comment:
How about define string as constants separately?

Reply:
I think define string as constants is meaningless, static variables are defined for reuse.

Reply:
Done, Thank you for your reminder